api/portainer.go

```diff
@@ -882,7 +883,12 @@ type (
 		// Portainer API version
 		Version string `json:"Version" example:"2.0.0"`
 		// Server Instance ID
-		InstanceID string `example:"299ab403-70a8-4c05-92f7-bf7a994d50df"`
+		InstanceID      string `example:"299ab403-70a8-4c05-92f7-bf7a994d50df"`
+		DemoEnvironment bool
+		DemoDetails     struct {
+			Users        []UserID
+			Environments []EndpointID
+		}
```

Comment:
```suggestion
		DemoEnvironment     struct {
		        Enabled    bool
			Users        []UserID
			Environments []EndpointID
		}
```
^ optionally could potentially have a single, non-nil struct that encapsulates the demo details/data.